PersonApi.Tests/V1/Controllers/PersonApiControllerTests.cs

```diff
@@ -11,14 +11,17 @@
 
 namespace PersonApi.Tests.V1.Controllers
 {
+    [Collection("LogCall collection")]
     public class PersonApiControllerTests
     {
+        private readonly LogCallAspectFixture _logCallFixture;
```

Comment:
is this used?

Reply:
I'll see if I can remove it and just leave the Collection attribute...